ddtrace/tracer.py

```diff
@@ -246,6 +248,25 @@ def current_trace_context(self, *args, **kwargs):
             return active.context
         return None
 
+    def get_log_correlation_context(self):
+        # type: () -> DDLogRecord
+        """Retrieves the Correlation Identifiers for the current active ``Trace``
+        This helper method generates a DDLogRecord for custom logging instrumentation including the trace id and
+        span id of the current active span, as well as the configured service, version, and environment names.
+        If there is no active span, an empty DDLogRecord will be returned.
```

Comment:
"empty" doesn't have any definition here - maybe something like "a DDLogRecord with all None values will be returned"?